batch/sql/case-sensitive-billing-project.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE billing_projects ADD COLUMN name_cs VARCHAR(100) COLLATE utf8mb4_0900_as_cs, ALGORITHM=INSTANT;
+ALTER TABLE billing_project_users ADD COLUMN user_cs VARCHAR(100) COLLATE utf8mb4_0900_as_cs, ALGORITHM=INSTANT;
+
+UPDATE billing_projects
+SET name_cs = `name`;
+
+UPDATE billing_project_users
+SET user_cs = user;
+
+ALTER TABLE billing_projects MODIFY COLUMN name_cs VARCHAR(100) NOT NULL COLLATE utf8mb4_0900_as_cs;
+ALTER TABLE billing_project_users MODIFY COLUMN user_cs VARCHAR(100) NOT NULL COLLATE utf8mb4_0900_as_cs;
+
+CREATE UNIQUE INDEX `billing_project_name_cs` ON `billing_projects` (`name_cs`);
+CREATE INDEX `billing_project_name_cs_status` ON `billing_projects` (`name_cs`, `status`);
+CREATE INDEX `billing_project_users_billing_project_user_cs` ON `billing_project_users` (`billing_project`, `user_cs`);
```

Comment:
What's the plan for dropping the old columns and fixing up the foreign key references?

Reply:
There is no plan to do so. I wrote out what needs to happen and it's complicated, is going to take a lot of work / testing to get right, and didn't seem as important as other things we could be doing. If you feel differently, then I can go ahead and do it.

Reply:
I forgot how far I got on this, but here was a pilot branch I believe: https://github.com/hail-is/hail/commit/32fe47906dc7c189703ba713cf0e3794084af475